ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/DownloadHeadersStep.java

```diff
@@ -36,7 +38,7 @@
 import org.slf4j.LoggerFactory;
 
 public class DownloadHeadersStep
-    implements Function<CheckpointRange, CompletableFuture<CheckpointRangeHeaders>> {
+    implements Function<SyncTargetRange, CompletableFuture<RangeHeaders>> {
```

Comment:
:+1:  next we will have to excise `fast` from fastSyncDownloader. ;)

Reply:
clearly Jiri's work will be good for this part of the code. it's starting to be complex to adapt this code to all of this synchronization options